src/Foundatio.Jobs.Commands/Foundatio.Jobs.Commands.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>net46</TargetFrameworks>
     <PackageTags>Distributed;Jobs;NETSTANDARD;Core</PackageTags>
   </PropertyGroup>
+  <PropertyGroup Label="Signing">
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>..\Exceptionless.snk</AssemblyOriginatorKeyFile>
```

Comment:
Are you sure it's not being copied over?  The reason I ask is that it is included in the [fork I've been working off of](https://github.com/TicketVista/Foundatio/tree/master/src) and it looks like it's included in the PR in the file above this file we're commenting on.